composer.json

```diff
@@ -9,5 +9,21 @@
             "email": "ezsystems@users.noreply.github.com"
         }
     ],
-    "require": {}
+    "require": {},
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^2.16"
```

Comment:
I believe we should still have dependency on php-cs-fixer in all package repositories because they will carry `.php_cs` file: https://github.com/ezsystems/ezplatform-page-builder/commit/9ef0f24b0a102699c6805d34e7b747bd4bd18fd4

Reply:
I'm against having hidden dependencies. If you use `php-cs-fixer` to maintain code style in your package then it's your dependency and should be a part of package's `composer.json`. I agree on moving `php-cs-fixer` from `require-dev` to `require` - this is indeed my mistake.